c/datasetjson.c

```diff
@@ -335,6 +339,28 @@ static int isPartionedDataset(char *dscb) {
 }
 #endif
 
+static int checkForVSAM(char* dsPath) {
```

Comment:
Please, use bool instead of int, since it returns TRUE or FALSE. Also, the function name should ideally be boolean-esque such as, isDatasetVSAM().